src/components/Api.js

```diff
@@ -12,9 +12,7 @@ export default class Api {
 
     getUserInfo() {
         return fetch(`${this._baseUrl}/users/me`, {
-          headers: {
-            authorization: "f01bb77e-1c08-4def-8c31-263c2557aed9"
-          }
+          headers: this._headers,
         }).then(this._checkServerResponse);  
       }
     
```

Comment:
` authorization: "f01bb77e-1c08-4def-8c31-263c2557aed9"` in line 22 should be replaced with ` headers: this._headers,` as well